src/main/java/com/ncc/aif/ValidateAIF.java

```diff
@@ -520,6 +563,19 @@ private static ReturnCode displaySummary(int fileCount, int invalidCount, int sk
         }
     }
 
+    /**
+     * Tells the validator to "fail fast" if validation errors are detected.  Validation will terminate after
+     * [abortThreshold] validation errors are detected.  Use zero to disable failing fast.
```

Comment:
Is `[]` the way I should  be referencing method parameters within JavaDoc? I've been using `<code></code>`

Reply:
Heh, no...`<code></code>` is correct.  The brackets were used by Ryan in AIFUtils and actually do something in Kotlin ([source](https://kotlinlang.org/docs/reference/kotlin-doc.html)).  This should probably be changed in AIFUtils also.

Reply:
Dang...I was kinda hoping there was a more terse form.

Reply:
Replaced all occurrences of `[variable]` with `<code>variable</code>`